RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -20,12 +20,16 @@
 
 from RecoEgamma.EgammaIsolationAlgos.particleBasedIsoProducer_cff import *
 
+from RecoJets.JetProducers.fixedGridRhoProducerFastjet_cfi import *
+fixedGridRhoFastjetAllTmp = fixedGridRhoFastjetAll.clone(pfCandidatesTag = cms.InputTag("particleFlowTmp"))
```

Comment:
RECO/AOD event content now does not have wildcards and you should add needed products explicitly.
... miniAOD event content, however still has wildcards and this fixedGridRhoFastjetAllTmp shows up in miniAOD files.